lib/sentry/event.ex

```diff
@@ -52,15 +54,33 @@ defmodule Sentry.Event do
       |> String.trim("*")
       |> String.trim
 
+    release = case Application.fetch_env(:sentry, :release) do
```

Comment:
I think we can use `Application.get_env(:sentry, _)`.  should return nil if not set